lua/README.md

```diff
@@ -0,0 +1,11 @@
+# mLRS - lua #
+
+This folder contains lua scripts which may be used to change the Tx module and connected receiver settings when the Tx module is installed in a OpenTx or EdgeTx radio.
+
+[mLRS-bw.lua](mLRS-bw.lua) is for radios with black and white screens
```

Comment:
I woud suggest to change the order of sequence

Reply:
You mean the order the files appear in?  I matched the order on github, but OK.  I guess the color screens are more important?

Reply:
yes :)